canonical-web-and-design/microk8s.io
# Remove the bespoke docs styling

## Done
Remove the bespoke docs styling and replace with Vanilla docs layout

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/docs
- Check it looks ok
